airbyte-integrations/connectors/source-greenhouse/integration_tests/acceptance.py

```diff
@@ -24,25 +24,11 @@
 
 
 import pytest
-from grnhse.exceptions import EndpointNotFound, HTTPError
-from source_greenhouse.client import Client
 
+pytest_plugins = ("source_acceptance_test.plugin",)
 
-def test__heal_check_with_wrong_api_key():
```

Comment:
Those tests are not valid, or why do we removed them?

Reply:
these tests are replaced by SAT